### ISSUE TITLE
`git-gitlab` forumula submitted to Homebrew

### ISSUE THREAD

**Issue by jwhitley** [numa08/git-gitlab#50]
This is a heads-up that I've submitted a `git-gitlab` formula to Homebrew, see Homebrew/homebrew-core#1064.  If/when that lands, it might be nice to update the README.md docs here to reflect the easier installation process for OS X users.

**Comment by jwhitley** [comment]
@numa08 Would you be amenable to tagging a release for this repo?  I'd neglected that Homebrew requires tagged upstream releases for formula submissions.

**Comment by asfaltboy** [comment]
@numa08 any chance for a release tag? E.g merge develop into master and freeze current to 0.2 ?

**Comment by jwhitley** [comment]
@asfaltboy  It's worth noting that many of the documented features in the README here are unimplemented, or somewhat buggy in their current state.  Kudos to @numa08, as it's a welcome start, but this project needs some work before it's really ready to be published in any packaging system.  Using Go for this is nice, since it would be possible to package and deploy a single binary for any given platform.  That makes distributing it to a team easier.

So far, the best solution I've found is [git-repo](https://github.com/guyzmo/git-repo), which supports GitHub, GitLab, and BitBucket.  `pip install git-repo`, then follow the configuration instructions in that projects' README.md.

Closing this unless/until the project matures a bit more.
